backend.go

```diff
@@ -135,6 +139,26 @@ func VersionedKVFactory(ctx context.Context, conf *logical.BackendConfig) (logic
 	return b, nil
 }
 
+func pathInvalid(b *versionedKVBackend) []*framework.Path {
+	handler := func(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+		resp := &logical.Response{}
+		resp.AddWarning("Invalid path for a versioned K/V store. See the API docs for the appropriate endpoints.")
+		return resp, nil
```

Comment:
This isn't currently returning a 404 since the UI is not setup to output warnings on 404. Can be enhanced though.

Reply:
I think we should ideally 404 with the warning. @meirish how bad would it be to support that?

Reply:
I think it should just work - if there’s any warning in any response we show that message in a blue flash message and I think 404s go through the same path.

Reply:
Though with the header, when would the UI use unsupported endpoints?